javascript/app.js

```diff
@@ -0,0 +1,336 @@
+		console.log("FizzBuzz");
+	} else if (i % 5 === 0) {
+		console.log("Buzz");
+	} else if (i % 3 === 0) {
+		console.log("Fizz");
+	} else console.log(i);
+};
+
+
+// GETTING TO KNOW YOU
+// Use the following arrays to answer the questions below (name, age, hometown):
+const shahzad = ["Shahzad", 1000, "Austin"];
+const jim = ["Jim", 16, "All cities"];
+const reuben = ["Reuben", 22, "Durham"];
+const daniel = ["Daniel", 186, "Dallas"];
+const ryan = ["Ryan", 65, "Denver"];
+
+// Daniel decides that Shahzad can't be named "Shahzad" anymore. Remove "Shahzad" from the shahzad array and replace it with "Gameboy".
+for (var i = 0; i < shahzad.length; i++) {
+	if (shahzad[i] === "Shahzad") {
+		shahzad[i] = "Gameboy";
+	}
+}
+console.log(shahzad);
+
+// Daniel just had his birthday; change Daniel's array to reflect him being a year older. Don't just hard code 187--pretend that you don't know that his age is 186, and write your code to just make him a year older than whatever age he was.
+daniel[1]++;
+console.log(daniel);
+
+// Ryan is Batman maybe. Or possibly Robin. Change Ryan's hometown from "Denver" to "Gotham City".
+ryan[2] = "Gotham City";
+console.log(ryan);
+
+// Reuben left Durham 5 years ago to come to Chicago. Remove "Durham" from Reuben's array and add "Chicago". (Note: remove and then add is different from simply changing the value at that index.)
+reuben.pop();
+reuben.push("Chicago");
+console.log(reuben);
+
+// Jim could be literally anywhere in the world. Remove "All cities" from his array, then pick any 3 cities you like, and add them to Jim's array. If you did it in 3 lines of code that's fine, but see if you can do it in one line of code
+jim.pop();
+jim.push("Hakodate", "Bergerac", "Nice");
+console.log(jim);
+
+// Bonus: Whoops! Jim is actually only allowed to be in one of two cities. Whatever the first of the 3 cities for Jim is now, remove it from the array using .splice()
+jim.splice((jim.length - 3), 1);
+console.log(jim);
+
+
+// YELL AT THE NINJA TURTLES
+// 1. Create an array with the members of the ninja turtles (Donatello, Leonardo, Raphael, Michaelangelo)
+let ninja = ['Donatello', 'Leonardo', 'Raphael', 'Michaelangelo'];
+
+// 2. Use a for loop to call .toUpperCase() on each of them and print out the result.
+for (var i = 0; i < ninja.length; i++) {
+	console.log(ninja[i].toUpperCase());
+};
+
+// 3. Bonus: Modify the answer you just wrote. Instead of all letters being uppercase, make the letters alternate back and forth between uppercase and lowercase.
+
+// look at each name index // for loop
+for (var i = 0; i < ninja.length; i++) {
+	// break name into single letters
+	let breakApart = [...ninja[i]];
+	// test break
+	// console.log(breakApart);
+	// look at name's single letters 
+	for (var j = 0; j < breakApart.length; j++) {
+		// every other letter
+		if (j % 2 === 0) {
+			// make uppercase
+			breakApart[j] = breakApart[j].toUpperCase();
+		}
+	}
+	// test uppercase
+	console.log(breakApart.join(''));
+}
```

Comment:
Cool, good job!  if you're using spread operator (`...`) make sure you understand how it works